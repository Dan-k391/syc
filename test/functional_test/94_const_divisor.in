0
3
-3
5
-5
6
-6
9
-9
118
-118
65534
-65534
65569
-65569
2147483647
-2147483647
