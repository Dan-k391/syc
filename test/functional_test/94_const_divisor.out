0
0
0
0
0
0
1
-1
0
0
0
0
-1
1
0
0
0
0
1
-1
0
0
0
0
-1
1
0
0
0
0
2
-2
0
0
0
0
-2
2
0
0
0
0
3
-3
0
0
0
0
-3
3
0
0
0
0
39
-39
0
0
0
0
-39
39
0
0
0
0
21844
-21844
0
0
0
0
-21844
21844
0
0
0
0
21856
-21856
1
-1
0
0
-21856
21856
-1
1
0
0
715827882
-715827882
32767
-32767
1
-1
-715827882
715827882
-32767
32767
-1
1
0